src/transform/inverse.rs

```diff
@@ -1492,7 +1492,9 @@ fn av1_idct64(input: &[i32], output: &mut [i32], range: usize) {
   output[63] = clamp_value(stg10[0] - stg10[63], range);
 }
 
-static INV_TXFM_FNS: [[fn(&[i32], &mut [i32], usize); 5]; 4] = [
+type InvTxfmFn = [fn(&[i32], &mut [i32], usize); 5];
```

Comment:
```suggestion
type InvTxfmFn = fn(&[i32], &mut [i32], usize)
```

would be a better way to break it apart I think